### ISSUE TITLE
Published npm package contains .git folder

### ISSUE THREAD

**Issue by trevorharwell** [rauldeheer/use-async-effect#6]
I cannot use npm at all because of this package.  I get this error:

```
npm ERR! path /Users/admin/project/node_modules/use-async-effect
npm ERR! code EISGIT
npm ERR! git /Users/admin/project/node_modules/use-async-effect: Appears to be a git repo or submodule.
npm ERR! git    /Users/admin/project/node_modules/use-async-effect
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

Sure enough running `ls -la ./node_modules/use-async-effect` lists the following:

```
total 48
drwxr-xr-x    10 admin  staff    320 Jul  8 15:50 .
drwxr-xr-x  1228 admin  staff  39296 Jul  8 16:06 ..
-rw-r--r--     1 admin  staff     89 Oct 26  1985 .flowconfig
drwxr-xr-x     3 admin  staff     96 Jul  8 15:50 .git
-rw-r--r--     1 admin  staff   1078 Oct 26  1985 LICENSE
-rw-r--r--     1 admin  staff    711 Oct 26  1985 README.md
-rw-r--r--     1 admin  staff    358 Oct 26  1985 index.js
-rw-r--r--     1 admin  staff    248 Oct 26  1985 index.js.flow
-rw-r--r--     1 admin  staff   1756 Jul  8 15:50 package.json
drwxr-xr-x     5 admin  staff    160 Jul  8 15:50 types
```

**Comment by houfio** [comment]
This issue got fixed in version 2.1.1, thanks for the report!

**Comment by trevorharwell** [comment]
@houfio Thank you for the quick fix!
